Gopkg.toml

```diff
@@ -35,6 +35,10 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.13.1"
 
+[[override]]
+  name = "k8s.io/api/coordination/v1beta1"
+  version = "kubernetes-1.13.1"
```

Comment:
There is no tag or branch of the `api` project called kubernetes-1.13.1 and even if there were, the coordination api wasn't part of it, and thats not the version we have stored in the vendor directory.

Reply:
fixed to the following:

[[constraint]]
  name = "k8s.io/api/coordination/v1beta1"
  source="https://github.com/kubernetes/api.git"
  branch="master"
  version = "kubernetes-1.13.0"

Reply:
fixed that to: